packages/deployment/app/kes.override.js

```diff
@@ -401,35 +431,57 @@ class UpdatedKes extends Kes {
         resolve(JSON.parse(body).tag_name);
       });
     });
-  };
+  }
 
   /**
    * Determines which release version should be downloaded from
    * cumulus-message-adapter repository and then downloads that file.
    *
-   * @return {Promise}
+   * @returns {Promise} returns the path of the extracted message adapter
    */
   fetchMessageAdapter() {
-    const messageAdapterVersion = this.config.message_adapter_version;
-    const releaseDownloadBaseUrl = `https://github.com/${this.messageAdapterGitPath}/releases/download`;
-    const messageAdapterFilename = this.config.message_adapter_filename;
-
-    if (!messageAdapterVersion) {
-      return this.fetchLatestMessageAdapterRelease()
-        .then((latestReleaseVersion) => {
-          const releaseLocation = `${releaseDownloadBaseUrl}/${latestReleaseVersion}/${messageAdapterFilename}`;
-          return this.downloadZipfile(releaseLocation, messageAdapterFilename);
-        });
-    } else {
-      const releaseLocation = `${releaseDownloadBaseUrl}/${messageAdapterVersion}/${messageAdapterFilename}`;
-      return this.downloadZipfile(releaseLocation, messageAdapterFilename);
+    if (this.config.message_adapter_filename) {
```

Comment:
Since the only thing that happens when this is false is that an empty Promise is returned, you could return that here and un-indent the rest of the function.

`if (!this.config.message_adapter_filename) return Promise.resolve();`